### DISCUSSION TITLE
Not sure what went run? validation checks all passed

### DISCUSSION THREAD

**Discussion by chocolocked** [Lightning-AI/pytorch-lightning#8182] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
raise RuntimeError("Cowardly refusing to serialize non-leaf tensor which requires_grad, "
RuntimeError: Cowardly refusing to serialize non-leaf tensor which requires_grad, since autograd does not support crossing process boundaries.  If you just want to transfer the data, call detach() on the tensor before serializing (e.g., putting it on the queue).

**Comment by Borda** [comment]
@chocolocked could you please clarify what is the question here? :rabbit:
